mllib/src/main/scala/org/apache/spark/mllib/regression/Lasso.scala

```diff
@@ -85,7 +85,8 @@ object LassoModel extends Loader[LassoModel] {
  * See also the documentation for the precise formulation.
  */
 @Since("0.8.0")
-@deprecated("Use ml.regression.LinearRegression with elasticNetParam = 1.0", "2.0.0")
+@deprecated("Use ml.regression.LinearRegression with elasticNetParam = 1.0 and regParam = 0.01",
```

Comment:
Now that I read this, I'm afraid it will confuse users, who might think they *have* to use regParam = 0.01.  Can you please change this to say "Use ml.regression.LinearRegression with elasticNetParam = 1.0.  Note the default regParam is 0.01 for LassoWithSGD, but is 0.0 for LinearRegression."
Same elsewhere.  Thanks a lot!